10-validation.md

```diff
@@ -118,6 +118,23 @@ An [Azure Advisor Alert](https://docs.microsoft.com/azure/advisor/advisor-overvi
 1. Select _Alerts_, then _Manage Rule Alerts_.
 1. There is an alert called "AllAzureAdvisorAlert" that will be triggered based on new Azure Advisor alerts.
 
+## Validate Azure Container Registry Image Pulls
+
+If you configured your third-party images to be pulled from your Azure Container Registry vs public registries, you can validate that the container registry logs show `Pull` logs for your cluster when you applied your flux configuration.
```

Comment:
Ideally we could prescribe a step in here or just let it be a note/tip. 

Please let's not be it a blocker, and proceed based on your preference.

Reply:
Please discard this comment, I'm seeing we created specific steps just below.